fiftyone/utils/eval/activitynet.py

```
@@ -489,13 +490,15 @@ def _from_dict(cls, d, samples, config, eval_key, **kwargs):
         recall = d["recall"]
         iou_threshs = d["iou_threshs"]
         thresholds = d.get("thresholds", None)
+        full_classwise_AP = d["full_classwise_AP"]
```

Pylint (4.0.2)

[convention] Variable name "full_classwise_AP" doesn't conform to '[a-z_]([a-z0-9_]{0,30})$' pattern

(C0103)